libs/core/src/lib/list/list-navigation-item/list-navigation-item.component.ts

```diff
@@ -99,11 +117,18 @@ export class ListNavigationItemComponent implements AfterContentInit, AfterViewI
             this._iconComponent._navigationItemIcon = true;
         }
         this._innerText = this._text.elementRef.nativeElement.textContent ?? '';
+        this._ariaLevel = 1;
+        this._childItems?.forEach((item) => {
+            item._ariaLevel = 2;
+        });
```

Comment:
If there is a child inside the navigation item, will it be ariaLevel 1 always? What I am saying is that maybe ariaLevel can be by default 0 and in afterContentInit somehow we could always increase it by 1?

Reply:
Or it can inject parent `ListNavigationItemComponent` and then increase it's value by 1 and set that value for itself?